server/src/test/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationPluginTest.java

```diff
@@ -0,0 +1,58 @@
+package org.opentosca.toscana.plugins.cloudformation;
+
+import java.io.File;
+import java.util.Set;
+
+import org.opentosca.toscana.core.BaseUnitTest;
+import org.opentosca.toscana.core.plugin.PluginFileAccess;
+import org.opentosca.toscana.core.transformation.logging.Log;
+import org.opentosca.toscana.model.EffectiveModel;
+import org.opentosca.toscana.model.node.Compute;
+import org.opentosca.toscana.model.node.RootNode;
+import org.opentosca.toscana.model.visitor.VisitableNode;
+import org.opentosca.toscana.plugins.cloudformation.visitor.CloudFormationNodeVisitor;
+import org.opentosca.toscana.plugins.testdata.TestEffectiveModels;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+public class CloudFormationPluginTest extends BaseUnitTest {
```

Comment:
Please consider replacing this with a ModelTransformIntegration test like [this one](https://github.com/StuPro-TOSCAna/TOSCAna/blob/master/server/src/integration/java/org/opentosca/toscana/plugins/kubernetes/KubernetesLampIT.java). Most of the code can probaly be adopted.

Would not consider it a problem tough

Reply:
I added the transform test but keeping this for now